go/service/gregor.go

```diff
@@ -185,12 +185,12 @@ func (g *gregorHandler) PushHandler(handler libkb.GregorInBandMessageHandler) {
 // otherwise it will try all of them. gregorHandler needs to be locked when calling
 // this function.
 func (g *gregorHandler) replayInBandMessages(ctx context.Context, t time.Time,
-	handler libkb.GregorInBandMessageHandler) error {
+	handler libkb.GregorInBandMessageHandler) ([]gregor.InBandMessage, error) {
 	var msgs []gregor.InBandMessage
 	var err error
 	if msgs, err = g.gregorCli.StateMachineInBandMessagesSince(t); err != nil {
 		g.G().Log.Errorf("gregor handler: unable to fetch messages for reply: %s", err)
-		return err
+		return []gregor.InBandMessage{}, err
```

Comment:
Is that better? I guess I wasn't sure which was considered the best way (I had nil in there first)